core/src/styles/index.scss

```diff
@@ -25,3 +25,8 @@
 @use 'utils/typography';
 @use 'utils/visibility';
 @use 'utils/ratio';
+@use 'utils/toast';
+
+@import  '../../node_modules/toastify-js/src/toastify.css';
```

Comment:
use @use instead of import and place it in a new // -- Vendor section between Reset and Core